pkgs/by-name/vc/vcpkg/package.nix

```
@@ -41,6 +41,16 @@ stdenvNoCC.mkDerivation (finalAttrs: {
       ''}
 
       ln -s "$out/bin/vcpkg" "$out/share/vcpkg/vcpkg"
+
+      # Here's the code that Creates this json file in visual studio deployment and the code that reads it:
```

Visual Studio is a proper noun so it should be capitalized:
```suggestion
      # Here's the code that Creates this json file in Visual Studio deployment and the code that reads it:
```

---

@Jayman2000  Will the script that generate the automatic update still work if we switch to `builtin.fetchGit`? Such as these: #327790